<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="medium_gear_tray">
    <static>true</static>
    <allow_auto_disable>false</allow_auto_disable>

    <link name="base_link">
      <inertial auto="true"/>
      
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://demo_parts/medium_gear_tray/meshes/medium_gear_part_tray.stl</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1.0 1.0 1.0 1.0</ambient>
          <diffuse>1.0 1.0 1.0 1.0</diffuse>
        </material>
      </visual>

      <collision name="collision">
        <density>20000</density>
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://demo_parts/medium_gear_tray/meshes/medium_gear_part_tray.stl</uri>
          </mesh>
        </geometry>
        <max_contacts>5</max_contacts>
      </collision>
    </link>

    <link name="slot_1">
      <inertial auto="true"/>
      <collision name="collision">
        <density>1</density>
        <pose>-0.05 0.05 -0.012 0 0 0</pose>
        <geometry>
          <box>
            <size>0.02 0.02 0.0001</size>
          </box>
        </geometry>
      </collision>
    </link>

    <link name="slot_2">
      <inertial auto="true"/>
      <collision name="collision">
        <density>1</density>
        <pose>0.05 0.05 -0.012 0 0 0</pose>
        <geometry>
          <box>
            <size>0.02 0.02 0.0001</size>
          </box>
        </geometry>
      </collision>
    </link>

    <link name="slot_3">
      <inertial auto="true"/>
      <collision name="collision">
        <density>1</density>
        <pose>-0.05 -0.05 -0.012 0 0 0</pose>
        <geometry>
          <box>
            <size>0.02 0.02 0.0001</size>
          </box>
        </geometry>
      </collision>
    </link>

    <link name="slot_4">
      <inertial auto="true"/>
      <collision name="collision">
        <density>1</density>
        <pose>0.05 -0.05 -0.012 0 0 0</pose>
        <geometry>
          <box>
            <size>0.02 0.02 0.0001</size>
          </box>
        </geometry>
      </collision>
    </link>

    <joint name="slot_1_joint" type="fixed">
      <parent>base_link</parent>
      <child>slot_1</child>
    </joint>

    <joint name="slot_2_joint" type="fixed">
      <parent>base_link</parent>
      <child>slot_2</child>
    </joint>

    <joint name="slot_3_joint" type="fixed">
      <parent>base_link</parent>
      <child>slot_3</child>
    </joint>
    <joint name="slot_4_joint" type="fixed">
      <parent>base_link</parent>
      <child>slot_4</child>
    </joint>

  </model>
</sdf>